device.go

```diff
@@ -209,7 +209,7 @@ type Device struct {
 	//   integer
 	// Description:
 	//   Network connection type. Refer to List 5.22.
-	Connectiontype int8 `json:"connectiontype,omitempty"`
+	Connectiontype *int8 `json:"connectiontype,omitempty"`
```

Comment:
Please, rename this field to `ConnectionType`, let's make it CamelCase. We'll release this as a next major version anyway, so renaming field is fine.

Reply:
done.
thank you.